_posts/2012-04-18-app.markdown

```diff
@@ -164,17 +173,17 @@ Wie benutzen die Scaffold-Funktion von Rails, um einen ersten Baustein für unse
 **Coach:** Was ist Rails Scaffolding? (Erläutere den Befehl, den Namen des Models und der entsprechenden Datenbanktabelle, Namenskonventionen, Attribute und Typen, usw.) Was sind Migrationen, und wofür brauchen wir sie?
 
 {% highlight sh %}
-rails generate scaffold idea name:string description:text picture:string
+bundle exec rails generate scaffold idea name:string description:text picture:string
```

Comment:
In der nitrous Version hatten wir so zentrale Hinweise a la 'wenn in einem Tutorial bla steht, mache foo'. So mussten wir nicht alles einzelnen Stellen anfassen, was den Gruppen die ohne cloudservice arbeiten sicher nicht so recht wäre. Ausserdem schaffen wir es ja eh nicht, *alle* tutorials anzupassen sondern höchstens das erste. Bundle exec ist in sofern anders, als das es auch im nicht-cloud-service setup Sinn ergibt.

Ich denke, wir sollten so einen Hinweis dorthinpacken, schon allein weil wir nicht alle tutorials aktualisieren werden. Der Hinweis kann dann auch problemlos in die englische Version (das bundle exec kann da ja eher nicht ohne Diskussion mit dem Rest der Welt rein). Es würden dann jedenfalls deutsche und englische Version erstmal funktionieren.

Mittelfristig sollten sich natürlich englische und deutsche Version wieder annähern, um den Pflegeaufwand klein zu halten. Ob die englische Version dann das bundle exec an allen Stellen auch einbaut oder wir es bei uns wieder ausbauen (und uns wie früher nur auf den einen Hinweis verlassen), können wir dann aber nachträglich noch sehen, das muss nicht sofort entschieden werden.